app/export.go

```diff
@@ -181,7 +181,7 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 
 	// Iterate through validators by power descending, reset bond heights, and
 	// update bond intra-tx counters.
-	store := ctx.KVStore(app.keys[stakingtypes.StoreKey])
+	store := ctx.KVStore(app.GetKey(stakingtypes.StoreKey))
```

Comment:
The keys are not meant to be exposed anywhere. The GetKey function even has a warning saying it only exists for testing purposes.

Reply:
I'm not sure how accurate that comment is, and the two lines are identical. Gaia is using it in the same fashion.
https://github.com/cosmos/gaia/blob/93016bd616d721910387a7b6e114db7c889e71c4/app/export.go#L176

Reply:
> Our keys have never been used this way.
We have never extracted the keys out of app.go. That is also a function on SimApp.